### PR TITLE
Fix fakeTimers failing frontend test

### LINKED ISSUE #968: `performance.mark is not a function` Causing frontend tests to fail
Need help figuring this out @andrewtedi

### PR DESCRIPTION
**Pull request by andrewtedi**
An error occurred as of late in which npm run frontend-test would hang until time exceeded or the workflow was terminated. This presents problems when it comes to our repository passing checks. This PR attempts to solve this issue by fixing errors found when running npm run frontend-test.

See [sinonjs/fake-timers#136](https://github.com/sinonjs/fake-timers/issues/136)

Resolves #968

### CODE REVIEW

**Comment by evanugarte** [comment]
lets see if the build finishes

**Comment by evanugarte** [comment]
also we dont use travis

**Comment by andrewtedi** [comment]
Oh right, we stopped using it a while back.

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ jobs:
       uses: supercharge/mongodb-github-action@1.3.0
     - name: Use Node.js
       uses: actions/setup-node@v2
+    - run: npm -v
+    - run: node -v
```
Comment (evanugarte) [inline .github/workflows/node.js.yml:23]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "The SCE Website",
   "scripts": {
     "api-test": "mocha",
-    "frontend-test": "mocha --require @babel/register --require ignore-styles test/frontend/*.js",
+    "frontend-test": "mocha --require @babel/register --require ignore-styles test/frontend/*.js --exit",
```
Comment (evanugarte) [inline package.json:7]:
```suggestion
    "frontend-test": "mocha --require @babel/register --require ignore-styles test/frontend/*.js",
```

**Review by neelp03 — CHANGES_REQUESTED** [review]
I think something went wrong in your changes, the build had been running for 30+ minutes

**Review by allen-tran — APPROVED** [review]
so sickness!!! 🔛 🔝 🔜 💯

File: package.json (hunk 2 above)
Comment (evanugarte) [reply package.json:7]: hmm this didnt work. someone isnt calling `done` at the end of their tests...

**Review by neelp03 — APPROVED** [review]

**Pull request merged**